data/json/items/tool/cooking.json

```diff
@@ -17,6 +17,19 @@
     "//": "It would be nice if this could have a use action to thaw frozen stuff.  It's not hot enough to work as a hotplate.",
     "flags": [ "LEAK_DAM", "RADIOACTIVE", "DURABLE_MELEE" ]
   },
+  {
+    "id": "butchering_kit",
+    "name": { "str": "butchering kit", "str_pl": "butchering kits" },
+    "type": "TOOL",
+    "description": "This is a kit with all the specialized knifes you would need for deboning, skinning and preparing the cuts of meat from an animal.",
```

Comment:
Maybe reword to "This is a kit containing all the specialized knives you need to debone, skin, and prepare cuts of meat from an animal."

Reply:
Yea, that's better.